### PR TITLE
docs: Stating material-ui dependency in component

### LINKED ISSUE #11: Mention in the component description whether it depends on material-ui

### PR DESCRIPTION
**Pull request by vishakhanihore**
fixes #11

### CODE REVIEW

**Comment by vishakhanihore** [comment]
@helfi92 Please Review

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche Please Review

File: src/components/Code/index.jsx [hunk 1]
```diff
@@ -13,7 +13,8 @@ const validLanguage = (props, propName) => {
 };
 
 /**
- * Render children as syntax-highlighted monospace code.
+ * Render children as syntax-highlighted monospace code.<br>
+ * * The following component doesn't depend on `Material-UI`.
```
Comment (helfi92) [inline src/components/Code/index.jsx]: No need to add a comment if it's doesn't depend on material-ui.

File: src/components/CodeEditor/index.jsx [hunk 2]
```diff
@@ -18,7 +18,9 @@ import './json-lint';
     width: '100%',
   },
 })
-/** Render an editor */
+/** Render an editor. <br>
+ * * The following component depends on `Material-UI`.
+ */
```
Comment (helfi92) [inline src/components/CodeEditor/index.jsx]:
```diff
diff --git a/src/components/CodeEditor/index.jsx b/src/components/CodeEditor/index.jsx
index 4b4fdac..4658574 100644
--- a/src/components/CodeEditor/index.jsx
+++ b/src/components/CodeEditor/index.jsx
@@ -19,7 +19,8 @@ import './json-lint';
   },
 })
 /** Render an editor. <br>
- * * The following component depends on `Material-UI`.
+ *
+ * _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
  */
 export default class CodeEditor extends Component {
   static propTypes = {
```

Let's do the same change for the other files that require material-ui.

Screen Shot 2019-10-14 at 10 12 00 AM

Comment (vishakhanihore) [reply src/components/CodeEditor/index.jsx]: Looks nice

**Comment by vishakhanihore** [comment]
@helfi92 made the requested changes. Please review

File: src/components/Code/index.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ const validLanguage = (props, propName) => {
 };
 
 /**
- * Render children as syntax-highlighted monospace code.
+ * Render children as syntax-highlighted monospace code.<br>
```
Comment (helfi92) [inline src/components/Code/index.jsx]: Remove `<br>`.

File: src/components/CodeEditor/index.jsx [hunk 4]
```diff
@@ -18,7 +18,9 @@ import './json-lint';
     width: '100%',
   },
 })
-/** Render an editor */
+/** Render an editor. <br>
```
Comment (helfi92) [inline src/components/CodeEditor/index.jsx]: Remove ` <br>`. Add an empty newline instead. Same with the others.

**Comment by vishakhanihore** [comment]
@helfi92 Done, Please Review

File: src/components/CodeEditor/index.jsx [hunk 5]
```diff
@@ -18,7 +18,10 @@ import './json-lint';
     width: '100%',
   },
 })
-/** Render an editor */
+/** Render an editor.
+ *
+ * * _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
```
Comment (helfi92) [inline src/components/CodeEditor/index.jsx]: No need to add an extra "*".

```
* _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
```

A similar change is required for the others.

**Comment by helfi92** [comment]
Let me know when this is ready for review.

**Pull request merged**